plugin/trino-hive/src/main/java/io/trino/plugin/hive/coercions/TimestampCoercer.java

```diff
@@ -48,29 +48,6 @@ public final class TimestampCoercer
 
     private TimestampCoercer() {}
 
-    public static class ShortTimestampToVarcharCoercer
```

Comment:
I am a little bit missed, why it's safe to remove it here?
It's because of this change in HivePageSource:
```
if (fromType instanceof TimestampType timestampType && toType instanceof VarcharType varcharType) {
            // if (timestampType.isShort()) {
            //    return Optional.of(new ShortTimestampToVarcharCoercer(timestampType, varcharType));
            // }
            return Optional.of(new LongTimestampToVarcharCoercer(timestampType, varcharType));
        }
```

Reply:
Previously we were respecting the timestamp precision - so we need a coercer which would could have timestamp read as `long` or it could be as `LongTimestamp` - but this change would ensure that we would be reading the timestamp column as a NANOSECOND - so we don't need `ShortTimestampToVarcharCoercer`